### DISCUSSION TITLE
What SQL courses would you recommend for beginners?

### DISCUSSION THREAD

**Discussion by MCOUDERTCATA** [BryanJenksCommunity/FAQ#74] [Q&A] [answered]
### Question

What SQL courses would you recommend for beginners to start learning it? the idea is down the line to also use it with Dataview in Obsedian. 
Thanks

**Comment by tallguyjenks** [comment, marked as answer]
Dataview doesnt use SQL explicitly nor do I think the plan is to ever implement SQL for use the plugin. In my video about it I just noted on the structure of the dataview queries is pretty much the same as SQL. 

as for learning materials I don't have a specific course in mind because I actually learned SQL on the job and not from courses.

I did take this course a while back and it could be a good intro:

https://www.codecademy.com/learn/learn-sql
